docs/commands/super.md

```diff
@@ -759,7 +766,7 @@ For the _search_ test, we'll search for the string pattern
 ```
 in the field `payload.pull_request.body`
 and we'll just count the number of matches found.
-The number of matches is small (3) so the query performance is dominated
+The number of matches is small (2) so the query performance is dominated
```

Comment:
I happened to notice and fix this small error that had been there previously. It's actually the "Search+" test that returns 3 matches.